### DISCUSSION TITLE
Custom fonts

### DISCUSSION THREAD

**Discussion by creedmangrum** [dolanmiu/docx#1386] [Q&A]
Hello,

I love using this tool. It has been a lifesaver for me. 

One question I had, how can we use custom fonts like Adobe or Google fonts in the generated document? Is that possible?

**Comment by dolanmiu** [comment]
Thank you for you kind words!

Regarding custom fonts, it needs to be in a special format and be embedded in the document:

https://github.com/dolanmiu/docx/issues/239

If you can develop such feature and contribute it would be appreciated

**Reply by creedmangrum** [reply]
Thanks for the quick response! And thank you for pointing me to that discussion. I will follow there. It might be an interesting project to figure out. My startup uses docx to generate agreement templates for our clients. One in particular has requested the custom fonts so up until now I haven't needed it. The more they want it the sooner I will get to work on this ha!
